python/python/bystro/ancestry/tests/test_ancestry_types.py

```diff
@@ -0,0 +1,215 @@
+"""Test ancestry_types.py."""
+import pytest
```

Comment:
Should we add a test to check that the frozen attrs are indeed frozen?

Reply:
good catch!